applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_geo_steady_state_piping_element.cpp

```diff
@@ -0,0 +1,293 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "custom_elements/geo_steady_state_Pw_piping_element.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities.h"
+
+namespace
+{
+
+using namespace Kratos;
+
+PointerVector<Node> CreateNodes()
```

Comment:
There are some build issues due to this function having the same name as the one in the test_line_interface_element.cpp. Since pipelines use unity, cpp files will be clumped together, so we get multiple functions with the same name in one resulting cpp